### ISSUE TITLE
Some characters not displaying correct

### ISSUE THREAD

**Issue by slmjkdbtl** [gittup/tup#369]
running `tup` on a non-tup directory:
```
tup UH��H�.� usage: tup [args]
For information on Tupfiles and other commands, see the tup(1) man page.
No .tup directory found. Either create a Tupfile.ini file at the top of your project, or manually run 'tup init' there.
```

What are those characters? Tried different fonts and still got this.

**Comment by gittup** [comment]
Hmm, the version string is supposed to appear where those weird characters are. Did you build tup yourself? Or did you download a binary from gittup.org or package manager? It looks like something is messed up how the version number was generated and built into the executable. If you built it yourself, what revision are you on? (eg: 'git describe' or the like)

**Comment by slmjkdbtl** [comment]
@gittup I'm on macos and using homebrew to install, but I tried building myself with the master version (`tup v0.7.8-20-g1d2fd514`) and there's no display error

**Comment by gittup** [comment]
I believe this was fixed once v0.7.10 was published in homebrew. See also #384.
